app/pulp/app/models/progress.py

```diff
@@ -27,6 +27,8 @@ class ProgressReport(Model):
         total: (models.IntegerField) The total count of items to be handled by the ProgressBar
             (required)
         done (models.IntegerField): The count of items already processed. Defaults to 0.
+        suffix (models.TextField): Customazible way of how progress report will be rendered
+            More info can be found here https://github.com/verigak/progress
```

Comment:
Rewriting to active voice:    `See https://github.com/verigak/progress for more info.`